app/templates/components/-report/metrics-chart/summary.hbs

```diff
@@ -0,0 +1,5 @@
+{{#each metrics as |metric|}}
+  <div class="btn btn-text disabled">
+    {{metric.label}}: {{metric.total}}
```

Comment:
This should use the number format tool to prevent huge number lengths (example 1,500,000 should be rendered at 1.5m)

Reply:
👍🏾 
![image](https://user-images.githubusercontent.com/1778222/112690104-39e50800-8e49-11eb-91fd-01bf856234b1.png)